shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IAppInsightsCore.ts

```diff
@@ -72,6 +75,13 @@ export interface IAppInsightsCore extends IPerfManagerProvider {
      */
     removeNotificationListener?(listener: INotificationListener): void;
 
+    /**
+     * Add a telemetry processor to decorate or drop telemetry events.
+     * @param telemetryInitializer - The Telemetry Initializer function
+     * @returns - A ITelemetryInitializerHandler to enable the initializer to be removed
+     */
+    addTelemetryInitializer(telemetryInitializer: TelemetryInitializerFunction): ITelemetryInitializerHandler | void;
```

Comment:
This is moving the telemetry initializer to the Core so all Sku's will have the telemetry initializer functionality not just the Web Analytics.